kpi/deployment_backends/kobocat_backend.py

```diff
@@ -790,20 +781,22 @@ def set_has_kpi_hooks(self):
             })
 
     def set_validation_status(self,
-                              submission_pk: int,
-                              data: dict,
+                              submission_id: int,
                               user: 'auth.User',
+                              data: dict,
```

Comment:
> i was wondering why this parameter is called `data` when `get_validation_status()` calls it `params`.

Because I wanted to be consistent with `Request` . `params` for `GET` and `data` for `POST`. 
It seems that [I haven't be so consistent anyway](https://github.com/kobotoolbox/kpi/pull/3124#discussion_r674814011).

> a compromise could be to ignore the `get_`, `set_`, etc. prefixes when sorting, so that we alphabetize by the thing being acted upon (in this case, `validation_status`). let's discuss more when we have a call.

🤔  I hear you.

https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTNSbv0cJ9e2AMzUEg_RKwkAv1-Xjl2g62-ni4cWh7Jdh3-O7X2vSxwMFTJmV1U8B1Ngdw&usqp=CAU